### PR TITLE
Better default values

### PR DESCRIPTION
**Pull request by jswift-stfc**
PR to implement default values for a range of attributes/parameters. Feedback would be good for those attributes that do not have default values / on the default values that have been set.

### CODE REVIEW

File: src/gudrun_classes/sample.py [hunk 1]
```diff
@@ -107,35 +107,35 @@ def __init__(self):
...
         self.tempForNormalisationPC = 0.0
         self.totalCrossSectionSource = CrossSectionSource.TRANSMISSION
         self.crossSectionFilename = ""
-        self.sampleTweakFactor = 0.0
-        self.topHatW = 0.0
-        self.FTMode = FTModes.SUB_AVERAGE
+        self.sampleTweakFactor = 1.0
+        self.topHatW = 3.0
+        self.FTMode = FTModes.ABSOLUTE
         self.minRadFT = 0.0
```
Comment (trisyoungs) [inline src/gudrun_classes/sample.py:128]: Very much sample dependent, but can be given a starting value of 0.75,

File: src/gudrun_classes/container.py [hunk 2]
```diff
@@ -68,18 +68,18 @@ def __init__(self):
...
         self.geometry = Geometry.SameAsBeam
-        self.upstreamThickness = 0.0
-        self.downstreamThickness = 0.0
+        self.upstreamThickness = 0.1
+        self.downstreamThickness = 0.1
         self.angleOfRotation = 0.0
-        self.sampleWidth = 0.0
+        self.sampleWidth = 5.0
         self.innerRadius = 0.0
         self.outerRadius = 0.0
```
[the remark this answers is not in the document]
Comment (jswift-stfc) [reply src/gudrun_classes/container.py:76]: Issue #288 opened.

File: src/gudrun_classes/sample.py [hunk 3]
```diff
@@ -107,35 +107,35 @@ def __init__(self):
...
         self.outerRadius = 0.0
         self.sampleHeight = 0.0
-        self.density = 0.0
+        self.density = 0.1
         self.densityUnits = UnitsOfDensity.ATOMIC
         self.tempForNormalisationPC = 0.0
         self.totalCrossSectionSource = CrossSectionSource.TRANSMISSION
         self.crossSectionFilename = ""
-        self.sampleTweakFactor = 0.0
-        self.topHatW = 0.0
```
[the remark this answers is not in the document]
Comment (jswift-stfc) [reply src/gudrun_classes/sample.py:126]: Issue #289 opened.

File: src/gudrun_classes/sample.py (hunk 1 above)
Comment (jswift-stfc) [reply src/gudrun_classes/sample.py:128]: Is that for the tweak factor?

Comment (trisyoungs) [reply src/gudrun_classes/sample.py:128]: Eek - sorry, this comment got added in very much the wrong place!  That's for the minimum radius of the FT.

**Pull request merged**